devtools/conda-envs/openeye.yaml

```diff
@@ -17,10 +17,12 @@ dependencies:
   - openff-forcefields
   - smirnoff99Frosst
   - openff-units >=0.2
+  - openff-models
+  - openff-amber-ff-ports
   - openff-utilities >=0.1.5
-  - openff-interchange-base >=0.3.3
-  - openff-nagl >=0.2.2
-  - openff-nagl-models >=0.0.2
+  # openff-interchange-base >=0.3.3
+  # openff-nagl >=0.2.2
+  # openff-nagl-models >=0.0.2
```

Comment:
Am I correct in thinking that all the changes to this file would be reverted before merging?